src/main/java/gigaherz/toolbelt/slot/ExtensionSlotBelt.java

```diff
@@ -173,7 +173,10 @@ public void playerDeath(PlayerDropsEvent event)
             if (instance == null) return;
             if (!player.world.getGameRules().getBoolean("keepInventory") && !player.isSpectator())
             {
-                event.getDrops().add(player.dropItem(instance.getBelt().getContents(), true, false));
+                ItemStack belt = instance.getBelt().getContents();
+                if (belt != null) {
```

Comment:
If getContents returns null, it should be fixed to never return null, the null check here shouldn't be necessary.